ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -119,6 +127,15 @@ type RayJobSpec struct {
 	// If the RayCluster is already created, it will be deleted.
 	// In case of transition to false a new RayCluster will be created.
 	Suspend bool `json:"suspend,omitempty"`
+	// ManagedBy is used to indicate the controller or entity that manages a RayJob.
+	// The value must be either empty, 'ray.io/kuberay-operator' or 'kueue.x-k8s.io/multikueue'.
+	// The kuberay-operator reconciles a RayJob which doesn't have this field at all or
+	// the field value is the reserved string 'ray.io/kuberay-operator',
+	// but delegates reconciling the RayJob with 'kueue.x-k8s.io/multikueue' to the Kueue.
+	// The field is immutable.
+	// TBD: Immutability from k8s v1.28 and above, below it won't be - is this acceptable?
```

Comment:
@andrewsykim @kevin85421 question to you, as we can enforce immutability with CEL Validation, but only from k8s v1.28

Reply:
CEL based validation has been Beta and on by default for quite a while now I think, so I tihnk it's fine?

Reply:
cool, let's go with it